docs/modules/ROOT/pages/reference/nix-functions/runCachixDeploy.adoc

```diff
@@ -86,6 +86,17 @@ And then copy the secret over to your Hercules CI agent `secrets.json`.
 
 You can generate a value for _ACTIVATION-TOKEN_ by navigating to https://app.cachix.org/deploy[Cachix Deploy], creating a workspace if you haven't already done so and clicking btn:[Start Deployment].
 
+[[param-async]]
+=== `async`
+
+Default: `false`
+
+Whether to exit immediately.
```

Comment:
```suggestion
Whether to exit immediately instead of waiting for all deployments to finish. This is useful if you are deploying to a system that might be offline, for example, a laptop.
```